Hunting Queries/MultipleDataSources/PersistViaIFEORegistryKey.yaml

```diff
@@ -23,7 +23,7 @@ query: |
   (
   SecurityEvent
   | where EventID == 4657
-  | where ObjectName has "Image File Execution Options"
+  | where ObjectName has_all ("\REGISTRY\MACHINE, "Image File Execution Options")
```

Comment:
Yyou need double back slashes: "\\REGISTRY\\MACHINE". Sorry if I mislead you. Anyways, just this fix and we are fine

Reply:
Added, thank you!